### ISSUE TITLE
Add bookmarks feature

### ISSUE THREAD

**Issue by luigino** [flaviotordini/minitube#101]
(possibly related to #21)
Would it be feasible to have actual bookmarks for videos?
Sometimes there are long talks or documentaries that one watches when time allows it.

**Comment by flaviotordini** [comment]
Yes and it's actually a good proposal. How do you imagine the UI?

**Comment by luigino** [comment]
Perhaps an additional drop-down menu like in a browser, with a options to add/remove entries to the playlist, one to bookmark the current video and maybe one to add selected videos. 

Could probably do without an editor, but will have to add an internal search mechanism.
